tests/regression/test_csm.py

```diff
@@ -0,0 +1,220 @@
+import pytest
+
+from brownie import reverts, web3
+
+from utils.config import (
+    contracts,
+    ContractsLazyLoader,
+)
+from utils.dsm import UnvetArgs, to_bytes, set_single_guardian
+from utils.evm_script import encode_error
+from utils.test.csm_helpers import csm_add_node_operator, get_ea_member, csm_upload_keys
+from utils.test.deposits_helpers import fill_deposit_buffer
+from utils.test.helpers import ETH
+from utils.test.oracle_report_helpers import oracle_report
+from utils.test.staking_router_helpers import set_staking_module_status, StakingModuleStatus
+
+contracts: ContractsLazyLoader = contracts
+
+CSM_MODULE_ID = 3
```

Comment:
strong statement. could we get the id by module address?

Reply:
this is an expected mainnet statement. we expect that id for CSM is 3